### PR TITLE
Delete

### PR DESCRIPTION
**Pull request by DanielChen1234**
Cleaned up some old code. Same functionality as before with deleting the trello board cards.

### CODE REVIEW

File: client/src/components/cards/CardsIndex.jsx [hunk 1]
```diff
@@ -83,8 +84,13 @@ class CardsIndex extends Component {
     this.props.actions.votesActions.createVote(cardId);
   }
 
+  delete(cardId) {
+    this.props.actions.cardsActions.deleteCard(cardId)
+  }
+
   render() {
     const { classes } = this.props;
+    console.log(this.props.cards)
```
Comment (BlaneCordes) [inline client/src/components/cards/CardsIndex.jsx]: lets not merge debug code

File: client/src/reducers/cards.js [hunk 2]
```diff
@@ -13,4 +14,26 @@ export default createReducer(initialState, {
         Object.assign({}, state, {
             cards: payload
         }),
+    [DELETE_CARD]: (state, payload) =>
+    Object.assign({}, state, {
```
Comment (BlaneCordes) [inline client/src/reducers/cards.js]: this should be indented right?

File: client/src/reducers/cards.js [hunk 3]
```diff
@@ -13,4 +14,26 @@ export default createReducer(initialState, {
...
+        cards: state.cards.filter(card => card.id !== payload)
+    }),
 });
+
+// export default function (state = initialState, action){
```
Comment (BlaneCordes) [inline client/src/reducers/cards.js]: we need this commented out code?

**Comment by DanielChen1234** [comment]
You’re welcome! I’ll make the changes that you requested. Do you want me to merge once that’s done?

**Comment by BlaneCordes** [comment]
@DanielChen1234 yep sounds good

**Pull request merged**